.github/workflows/continuous-integration.yml

```diff
@@ -9,11 +13,51 @@ jobs:
     runs-on: windows-latest
 
     steps:
-    - name: Install opensim-core
+    - uses: actions/checkout@v2
+
+    - name: Install SWIG
+      run: choco install swig --version 3.0.12 --yes --limit-output
+      
+    - name: Install NumPy 
+      run: python -m pip install numpy
+      
+    - name: Checkout opensim-core master
+      uses: actions/checkout@v2
+      with:
+         name: opensim-org/opensim-core
+         repository: opensim-org/opensim-core
+         path: 'opensim-core'
+    
+    - name: Build opensim-core-dependencies
+      run: |
+        echo $env:GITHUB_WORKSPACE\\build_deps
+        mkdir $env:GITHUB_WORKSPACE\\build_deps
+        chdir $env:GITHUB_WORKSPACE\\build_deps
+        # gci -r $env:GITHUB_WORKSPACE\\opensim-core
+        # /W0 disables warnings. The other flags are copied from CMake's
+        # default CMAKE_CXX_FLAGS.
+        # https://msdn.microsoft.com/en-us/library/19z1t1wy.aspx
+        $env:CXXFLAGS = "/W0"
+        cmake $env:GITHUB_WORKSPACE/opensim-core/dependencies -G"Visual Studio 16 2019" -A x64 -DSUPERBUILD_ezc3d:BOOL=on -DCMAKE_INSTALL_PREFIX=~/opensim_dependencies_install
+        $env:CXXFLAGS = ""
+        cmake . -LAH
+        cmake --build . --config Release -- /maxcpucount:4 
+
+    - name: Build opensim-core
+      run: |
+        echo $env:GITHUB_WORKSPACE\\build_core
+        mkdir $env:GITHUB_WORKSPACE\\build_core
+        chdir $env:GITHUB_WORKSPACE\\build_core
+        $env:CXXFLAGS = "/W0"
+        cmake $env:GITHUB_WORKSPACE/opensim-core -G"Visual Studio 16 2019" -A x64 -DOPENSIM_DEPENDENCIES_DIR=~/opensim_dependencies_install -DBUILD_JAVA_WRAPPING=on -DBUILD_PYTHON_WRAPPING=on -DOPENSIM_C3D_PARSER=ezc3d -DBUILD_TESTING=off -DCMAKE_INSTALL_PREFIX=~/opensim-core-install
+        cmake . -LAH
+        cmake --build . --config Release -- /maxcpucount:8
```

Comment:
Any reason for using 8 here and 4 above?

Reply:
FYI The Actions CI runners only have 2 cores.

Reply:
Thanks @chrisdembia and @halleysfifthinc changed to 2